Update change-your-sharepoint-domain-name.md (#3332)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md

3. Review any third-party apps, custom apps, and scripts that access SharePoint. They might need to be modified to use the new domain.

> >[!IMPORTANT]
> > if you have custom SharePoint Framework solutions that require access to API, it's required to access the "API access" page in SharePoint Online tenant administration to ensure that the new domain name can be used by SharePoint Framework components.

Reviewer: ```suggestion
> [!IMPORTANT]
> If you have custom SharePoint Framework solutions that require access to an API, you should access the API access page in SharePoint Online tenant administration to ensure that the new domain name can be used by SharePoint Framework components.
```
